templates/5_rename_and_reorganize.ipynb

```diff
@@ -0,0 +1,211 @@
+{
```

Comment:
<p>one of these should be uncommented</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/angelolab/toffy/pull/131/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>